packages/core/test/stateIn.test.ts

```diff
@@ -14,11 +15,11 @@ const machine = Machine({
             },
             EVENT2: {
               target: 'a2',
-              in: { b: 'b2' }
+              cond: stateIn({ b: 'b2' })
```

Comment:
we can remove or at least deprecated `in:` now, right?

Reply:
Almost; we have to decide what we want to do with regard to "relative-by-grandparent" state nodes.

Reply:
See card; we'll deprecate `in:` in a separate PR https://github.com/davidkpiano/xstate/projects/1#card-40695477